capstone_project/src/main/java/com/google/sps/data/School.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
```

Comment:
Could you update this to 2020?

Reply:
I have a question about this. I understand leaving the copyright info re: Google and the Apache License for code that we used from the starter code, but since we are also writing our own classes, should that be Copyright 2020 -our name- ? Or since it's owned by Google anyways, do we need to maintain this header? And/or, do we even care and is it better to maintain consistency and include this everywhere?